src/transformers/models/fsmt/convert_fsmt_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -188,10 +188,12 @@ def convert_fsmt_checkpoint_to_pytorch(fsmt_checkpoint_path, pytorch_dump_folder
         "encoder_ffn_dim": args["encoder_ffn_embed_dim"],
         "encoder_layerdrop": args["encoder_layerdrop"],
         "encoder_layers": args["encoder_layers"],
+        "encoder_normalize_before": args["encoder_normalize_before"],
```

Comment:
```suggestion
        "encoder_normalize_before": args.get("encoder_normalize_before", False),
```
this and the one below entry will fail with the original fairseq, so we need to check the key exists and set to False by default. 

I think the above should do the trick, but please test.

Reply:
I agree with your suggestion :)